index.js

```diff
@@ -126,3 +126,44 @@ exports.misses = function() {
 exports.keys = function() {
   return Object.keys(cache);
 };
+
+exports.exportJson = function() {
+  var plainJsCache = {};
+
+  // Discard the `timeout` property.
+  // Note: JSON doesn't support `NaN`, so convert it to `'NaN'`.
+  for (var key in cache) {
+    var record = cache[key];
+    plainJsCache[key] = {
+      value: record.value,
+      expire: record.expire || 'NaN',
+    };
+  }
+
+  return JSON.stringify(plainJsCache);
+};
+
+exports.importJson = function(jsonToImport) {
+  var cacheToImport = JSON.parse(jsonToImport);
+  var currTime = Date.now();
+
+  for (var key in cacheToImport) {
+    if (cacheToImport.hasOwnProperty(key)) {
+      var record = cacheToImport[key];
+
+      // This could possibly be `'NaN'` if there's no expiry set.
+      var remainingTime = record.expire - currTime;
+
+      if (remainingTime <= 0) {
+        // Delete any record that might exist with the same key.
+        exports.del(key);
```

Comment:
Actually this is deleting the key rather than ignoring it. If you import a key that has expired, it will cause the key to be removed from the existing cache, as would have happened automatically.

It's a good idea to add an option to skip duplicates instead of overwriting -- I'll do that soon. I don't think throwing makes sense though, since it's a perfectly valid key, it's just supposed to have expired. Thoughts?